cluster-up/cluster/kind/common.sh

```diff
@@ -150,12 +150,12 @@ function _get_pods() {
 function _fix_node_labels() {
     # Due to inconsistent labels and taints state in multi-nodes clusters,
     # it is nessecery to remove taint NoSchedule and set role labels manualy:
-    #   Master nodes might lack 'scheduable=true' label and have NoScheduable taint.
+    #   Control-plane nodes might lack 'scheduable=true' label and have NoScheduable taint.
     #   Worker nodes might lack worker role label.
     master_nodes=$(_get_nodes | grep -i $MASTER_NODES_PATTERN | awk '{print $1}')
     for node in ${master_nodes[@]}; do
         # removing NoSchedule taint if is there
-        if _kubectl taint nodes $node node-role.kubernetes.io/master:NoSchedule-; then
+        if _kubectl taint nodes $node node-role.kubernetes.io/control-plane:NoSchedule-; then
```

Comment:
Thanks for the PR

About `node-role.kubernetes.io/master` (appears few times in the PR),
IIUC it is set by k8s `kubeadm init` itself when creating the cluster, so we can't replace reference to `node-role.kubernetes.io/control-plane`, unless we also set k8s itself to use it,
or at least rename this taint one time in the very beginning after the cluster is created ?

worth to add ok-to-test to let tests run

Reply:
It should not be an issue, per https://v1-20.docs.kubernetes.io/docs/setup/release/notes/#changelog-since-v1-19-0 the new label is added by default so no changes to scripts creating the cluster should be required.

Reply:
Please see the jobs, they failed to find this taint

Reply:
Thanks. I'll do some trying

Reply:
Had a second read of the docs and actually the new label is applied but the new taint has not been added yet, I guess we'd have to keep the old taint reference for the time being.

Reply:
Thanks @oshoval @acardace I force pushed the changes without `node-role.kubernetes.io/control-plane:NoSchedule`